src/backend/joanie/core/models/courses.py

```diff
@@ -223,7 +223,13 @@ def clean(self):
 
 class CourseProductRelation(BaseModel):
     """
-    CourseProductRelation model allows to define position of each product on a course.
+    CourseProductRelation model allows defining the products sold on a given course.
+
+    It is used as through model in a ManyToMany relation between a Course and a Product.
+
+    The `organizations` field defines what organizations are selling the product on the
+    course. If several organizations are selling a product on a course, the buyer is asked
+    to choose from which organization s.he wants to buy it.
```

Comment:
It seems that `they` is the right neutral pronounce in english
```suggestion
    to choose from which organization they want to buy it.
```